tests/rustworkx_tests/generators/test_ladder.py

```diff
@@ -0,0 +1,26 @@
+# Licensed under the Apache License, 
+
+import unittest
+
+import rustworkx
+
+
+class TestLadderGraph(unittest.TestCase):
+    def test_ladder_graph(self):
+        graph = rustworkx.generators.ladder_graph(20)
+        self.assertEqual(len(graph), 40)
```

Comment:
Is this correct? We run `ladder_graph(20)` and it returns a graph with 40 nodes?

Reply:
I believe so because `let mut graph = StablePyGraph::<Undirected>::with_capacity(node_len * 2, num_edges);`